sdk/purview/purview-administration-rest/README.md

```diff
@@ -161,7 +161,7 @@ If you'd like to contribute to this library, please read the [contributing guide
 
 [account_product_documentation]: https://azure.microsoft.com/services/purview/
 [rest_client]: https://github.com/Azure/azure-sdk-for-js/blob/main/documentation/rest-clients.md
-[source_code]: https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-catalog-rest
+[source_code]: https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-administration-rest
```

Comment:
```suggestion
[source_code]: https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-scanning-rest
```

Reply:
@qiaozha I think this is actually a bug likely from copy and paste. The link to source code of purview-administration-rest should point to https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/purview/purview-administration-rest

Reply:
ok